.github/workflows/std-release.yaml

```diff
@@ -9,7 +9,8 @@ jobs:
     if: startsWith(github.event.release.name, '@cardano-sdk/cardano-services@')
     uses: ./.github/workflows/std.yml
     with:
-      deploy-staging-preprod: true
+      # Only deploy staging-* on demand, otherwise they don’t differ from dev-*:
```

Comment:
```suggestion
      # staging-* is deployed, with the versions and services identical to dev-*, at the start of the Lace release process:
```

Reply:
Applied in 9a6727a61e287514b3349afc53f47dba42bd4b98 ([compare](https://github.com/input-output-hk/cardano-js-sdk/compare/e177883e406baafbac3aeb8802e8f8c2aa022e26..9a6727a61e287514b3349afc53f47dba42bd4b98))